third_party/airflow/armada/operators/armada.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         job_service_client: JobServiceClient,
         armada_queue: str,
         job_request_items,
-        lookout_url_template: str,
+        lookout_url_template: str = "",
```

Comment:
```suggestion
        lookout_url_template: Optional[str] = None,
```
This is typically the way of declaring types optional